modules/alerting/main.tf

```diff
@@ -672,7 +693,14 @@ resource "google_logging_metric" "cloudrun_timeout" {
   }
 }
 
+moved {
+  from = google_logging_metric.dockerhub_ratelimit
+  to   = google_logging_metric.dockerhub_ratelimit[0]
+}
+
 resource "google_logging_metric" "dockerhub_ratelimit" {
```

Comment:
dumb question: the rate limits are shared across all services, does it make sense to exclude those for squad labeled services?

Reply:
yes, this is what this is doing

if squad field is empty, implies global, set count to 1, create it
if squad field is non-empty, implies squad alert, set count to 0, dont create

due to the oddity of using count, i need to move the old resource state to the new one with the index